apps/client/src/app/pages/voyage-tracker/voyage-tracker/vessel-row/vessel-rank-column/vessel-rank-column.component.ts

```diff
@@ -0,0 +1,32 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { FreeCompanyWorkshopFacade } from '../../../../../modules/free-company-workshops/+state/free-company-workshop.facade';
+import { observeInput } from '../../../../../core/rxjs/observe-input';
+
+@Component({
+  selector: 'app-vessel-rank-column',
+  templateUrl: './vessel-rank-column.component.html',
+  styleUrls: ['./vessel-rank-column.component.less'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class VesselRankColumnComponent {
+  @Input() rank: number;
+
+  vessel$ = observeInput(this, 'rank');
```

Comment:
Don't observe input if you're not going to use the observable 😃 my guess is that you tried to replicate what you saw in other components (which is fine !) because as you said, you're not very familiar with Angular.